site/content/en/guides/consumer/function/_index.md

```diff
@@ -1,18 +1,24 @@
 ---
-title: "Running functions"
+title: "Running Functions"
 linkTitle: "Running Functions"
 weight: 7
 type: docs
 description: >
     Modify or validate the contents of a package by calling a function.
 ---
 
-## Functions User Guide
+## Functions Explained
 
 When an operation requires more than just the schema, and data is necessary,
```

Comment:
"When an operation requires more than just the schema, and data is necessary" not sure what this means. 

Refer to what used to be into page in SDK repo for introduction of functions.

Reply:
Rewrote the section using previous SDK page